
{% extends "layout.html" %}
  
{% block title %}Experiments{% endblock %}
{% block lead %}Experiments.{% endblock %}

{% block page_content %}

	<!-- Page Content -->




	<div class="inner-banner inner-bg3 header-mask"
		style="background: url(https://static.igem.wiki/teams/5210/photo/experiment.jpg) no-repeat center 0% fixed;width: 100%;background-size: 100%;height: 96vh;">
		<div class="container">
			<div class="inner-title">
				<h3>Experiments</h3>
			</div>
			<!----scroll icon------->
			<div class="wrapper">
				<div class="scroll-hint">
					<p style="text-indent: unset;">Scroll To Explore</p>
					<div class="icon-scroll">
					</div>
				</div>
			</div>
		</div>
	</div>







	<!-- Page Content  请大家编辑以下部分 -->

	<div class="container relative">

		<!-------这个是膜态框的内容，注意这个一定要放在boxcontainer的外面，不然会出bug-------->

		<!--------模态图片 这个是通用的，所有图片共用这一个div-------->
		<div id="modal01" class="w3-modal" onclick="this.style.display='none'">
			<br>
			<div class="w3-modal-content w3-animate-zoom">
				<img id="img01" style="width:70%">
			</div>
		</div>







		<div class="tabsss" style="margin-top: 20px;">
			<input type="radio" id="radio-1" name="tabsss" checked=""
				onclick="javascript:document.getElementById('d1').style.display='block';document.getElementById('d2').style.display='none'">
			<label class="tabbb" for="radio-1" style="overflow: hidden;">Protocols</label>
			<input type="radio" id="radio-2" name="tabsss"
				onclick="javascript:document.getElementById('d1').style.display='none';document.getElementById('d2').style.display='block'">
			<label class="tabbb" for="radio-2">Experiment results</label>
			<span class="glider"></span>
		</div>







		<div id="d1">


			<!-- 一 -->
			<div id="references1" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d1" aria-expanded="false"
						aria-controls="references-d1" onclick="myFunction(this)">
						<h1 class="accordion-general-label">1. Configuration of LB medium</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d1" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">

							<p>Prepare 200 ml of medium by weighing 2 g of tryptone, 1 g of yeast extract, 2 g of NaCl, and 3 g of agar powder, adding 200 ml of purified water, and stirring with a glass rod until the solutes are dissolved. The pH was adjusted to 7.0-7.4 with 5 mol/L NaOH (usually 40 μl), and then the mixture was dispensed into 250 ml conical flasks. The conical flask was sealed with a sealing film and secured with a rubber band. Then steam sterilize the mixture under high pressure at 121°C, 103.4 kPa for 20 min. remove it, place it on an ultra-clean bench, cool it down to 50~55°C, and then add the antibiotic (kanamycin) to a final working concentration of 50 μg/ml. </p>

						</div>
					</div>
				</div>
			</div>

			<!-- 二 -->
			<div id="references2" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d2" aria-expanded="false"
						aria-controls="references-d2" onclick="myFunction(this)">
						<h1 class="accordion-general-label">2. PCR amplification</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d2" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">

							<p>Take the ice box and put dNTP, PrimeSTAR enzyme, 5X PrimeSTAR buffer, upstream and downstream primers, template plasmid, and dd water, into the ice box, and leave it to melt. Take the corresponding number of microtubes and place them in the ice box. Configure the reaction system according to primestar. Note the amount of template used (concentration determines volume)
							</p>
							<table class="model">
								<tbody>
									<tr style="background: rgb(33, 115, 192)">
										<th>Reaction System Composition</th>
										<th>Spiked volume</th>
										<th>Final concentration</th>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>5 X PrimeSTAR Buffer (Mg<sup>2+</sup>Plus)</td>
										<td>10μl</td>
										<td>1 X</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
									<td>dNTP Mixture (2.5mM each)</td>
									<td>4μl</td>
									<td>200μM each</td>

									<tr style="background: rgb(255, 255, 255)">
										<td>Primer1</td>
										<td>10-15pmol</td>
										<td>0.2-0.3μM</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>Primer2</td>
										<td>10-15pmol</td>
										<td>0.2-0.3μM</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>Template</td>
										<td><200ng</td>
										<td></td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>PrimeSTAR HS DNA Polymerase (2.5U/μl)</td>
										<td>0.5ul</td>
										<td>1.25units/50μl</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>DdH<sub>2</sub>O</td>
										<td>Up to 50 μl</td>
										<td></td>
									</tr>

								</tbody>
							</table>
							<p>Note: The total volume is 50 μL</p>
							<p>Turn on the PCR instrument and set up the program according to the reaction conditions.
							</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 三 -->
			<div id="references3" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d3" aria-expanded="false"
						aria-controls="references-d3" onclick="myFunction(this)">
						<h1 class="accordion-general-label">3. Electrophoretic verification of PCR results</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d3" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">

							<p>Preparation of gel: weigh 0.35g agar powder, add it into 30ml 1X TAE (Tris-acetic acid electrophoresis buffer) flask, microwave heating for 2.5min, and cool it down to 50 ℃. Remove the gel from the 4 ℃ refrigerator and aspirate 3 ml with a special gun for electrophoresis, stick the gun tip under the liquid surface and rotate it slightly to mix it well. Pour the liquid into the grid for dispensing the gel, insert the comb and wait for solidification. Sample Preparation: After PCR is finished, remove each reaction tube, take 5μl of each and add it to a new microtube, label it, add 1ul of 6X Sampling Buffer to each and mix well.   

							</p>
							<p>Run the gel: take out the Marker from -20℃ refrigerator, pull out the comb from the solidified gel, put it into the electrophoresis tank according to the direction, fill with 1×TAE add the prepared samples into each gel well and add 5ul of melted Marker, record the order of the samples in the wells. Run the gel at 150V constant pressure. Observe under UV light and compare the position of the fluorescent bands with the labeling instructions. Take photos.
							</p>


						</div>
					</div>
				</div>
			</div>

			<!-- 四 -->
			<div id="references4" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d4" aria-expanded="false"
						aria-controls="references-d4" onclick="myFunction(this)">
						<h1 class="accordion-general-label">4. DNA purification and recovery</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d4" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>We adopt the faster column purification and recovery method, and use the universal DNA purification and recovery kit (DP214) from TIANGEN Biochemical Technology, which mainly recovers DNA from agarose gel.
							</p>
							<p><b>Cut the gel:</b> use a clean blade to cut off the target DNA bands, try not to cut more than one, put it into a 1.5 mL ep tube (cut the gel quickly, avoid UV light), weigh the total weight and the weight of the empty 1.5 ml ep tube to calculate the weight of the gel block.
							</p>

			
							<p><b>Solubilization:</b> Add an equal volume of solution PC to the gel block for solubilization (100 μl of PC solution for every 0.1 g of gel block), and water bath at 50℃ for 10 min, during which the centrifuge tube is constantly and gently turned up and down to ensure that the gel block is completely solubilized.
							</p>
							<p><b>Column equilibrium:</b> put the adsorption column CB2 of the kit into the collection tube, add 500 μl of equilibrium solution BL, centrifuge at 12,000 rpm for 1 min, pour out the waste liquid in the collection tube, and adsorption column CB2 into the collection tube.
							</p>
							<p><b>Adsorption:</b> Add the solution obtained from the lysate to the adsorption column CB2 in the collection tube, centrifuge at 12,000 rpm for 1 min, pour out the waste liquid in the collection tube, and put the adsorption column CB2 into the collection tube.
							</p>
							<p><b>Wash:</b> Add 600 μl of rinse solution PW to the adsorbent column CB2, centrifuge at 12,000 rpm for 1 min, pour out the waste liquid from the collection tube, and put the adsorbent column CB2 into the collection tube. If high purity is required, the column should be washed again with the same procedure.</p>
							<p><b>Drying:</b> The column CB2 is centrifuged at 12,000 rpm for 2 min to remove as much of the rinse solution as possible. Leave the column at room temperature for a few minutes and then dry thoroughly. (The rinse solution contains ethanol, and ethanol residues can interfere with subsequent enzyme reaction experiments.)

							</p>
							
							<p><b>Elution:</b> Place the adsorbent column CB2 into a clean centrifuge tube and add an appropriate amount of elution buffer EB to the center of the adsorbent membrane (if the recovered target fragment is >4kb, the elution buffer EB should be heated in a water bath at 65-70℃) and leave it at room temperature for 2 min. Centrifuge the adsorbent column at 12,000rpm for 2 min and collect the DNA solution. After measuring the DNA concentration, store the DNA solution at -20℃.
							</p>

						</div>
					</div>
				</div>
			</div>

			<!-- 五 -->
			<div id="references5" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d5" aria-expanded="false"
						aria-controls="references-d5" onclick="myFunction(this)">
						<h1 class="accordion-general-label">5. Enzymatic digestion of PCR products</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d5" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>Use Thermo Fisher Scientific's FastDigest for enzyme digestion, take EcoR Ⅰ and Hind Ⅲ enzyme digestion sites for example：</p>

							<div class="row mx-auto d-flex justify-content-center wi-lg-100">
								<img src="https://static.igem.wiki/teams/5210/protocol-5.png" alt="Gut-sweeper"
								style="width:80%;cursor:pointer;">
							</div>

							<p>Our enzymatic system is presented below:</p>

							<table class="model">
								<tbody>

									<tr style="background: rgb(33, 115, 192)">
										<th>Reaction System Composition</th>
										<th>Spiked volume</th>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>DdH<sub>2</sub>O</td>
										<td>24μl</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>FastDigest enzyme(EcoR Ⅰ)</td>
										<td>1μl</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>FastDigest enzyme(Hind Ⅲ)</td>
										<td>1μl</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>
											10×FastDigest Buffer <br>
											Or 10×FastDigest Green Buffer
										</td>
										<td>4μl</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>DNA(Plasmid DNA/PCR product)</td>
										<td>2μl/10μl</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>Total volume</td>
										<td>32μl/40μl</td>
									</tr>
								</tbody>
							</table>

							<p>Incubate at 37℃ in water bath for 15min, heat at 80℃ for 10min to terminate the enzyme digestion, recover the DNA product.</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 六 -->
			<div id="references6" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d6" aria-expanded="false"
						aria-controls="references-d6" onclick="myFunction(this)">
						<h1 class="accordion-general-label">6. Plasmid introduction of fragment carriers
						</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d6" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>The appropriate molar ratio of plasmid vector to target gene fragment is 1:4 to 1:3, while the required volume ratio is calculated based on concentration and fragment length.
							</p>
							<p>The reaction needs to be carried out on ice and the reaction system is as follows:
							</p>
							
							<table class="model">
								<tbody>
									<tr style="background: rgb(33, 115, 192)">
										<th>Reaction System Composition</th>
										<th>Spiked volume</th>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>Plasmid gel recovery products</td>
										<td>2μl</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>PCR digestion of recovered products</td>
										<td>8μl</td>
									</tr>
								</tbody>
							</table>
						</div>
					</div>
				</div>
			</div>

			<!-- 七 -->
			<div id="references7" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d7" aria-expanded="false"
						aria-controls="references-d7" onclick="myFunction(this)">
						<h1 class="accordion-general-label">7. Plasmid extraction:TIANGEN Plasmid Extraction Kit (DP103)</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d7" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>1. Place the adsorption column CP3 into the collection tube, add 500 μl of equilibrium solution BL to the adsorption column, centrifuge at 12,000 rpm for 1 min, pour out the waste liquid from the collection tube, and place the adsorption column back into the collection tube.
							</p>
							<p>2. Take 1-5 ml of the overnight culture, add it to the centrifuge tube, centrifuge at 12,000 rpm for 1 min using a conventional tabletop centrifuge, and aspirate the supernatant as much as possible.
							</p>
							<p>3. Add 150 μl of Solution P1 (with RNaseA added) to the centrifuge tube with the bacterial precipitate and thoroughly suspend the bacterial precipitate using a pipette or vortex shaker.
							</p>
							<p>4. Add 150 μl of Solution P2 to the centrifuge tube and gently spin up and down 6-8 times to completely lyse the organism. Due to the use of TIANRed, the color of the solution will be a clear purple when Solution P2 is added and mixed thoroughly. If there is a mixture of cloudy red and purple colors, the lysis is not complete. Continue mixing until the color of the solution is completely clear purple.
							</p>
							<p>5. Add 350 μl of Solution P5 to a centrifuge tube and immediately mix rapidly up and down 12-20 times; when well mixed, a flocculated precipitate will appear. Centrifuge at 12,000 rpm for 2 min.
							</p>
							<p>6. Transfer the supernatant collected in the previous step to the adsorbent column CP3 using a pipette (the column is placed in the collection tube), taking care not to aspirate the precipitate as much as possible. Centrifuge the column at 12,000 rpm for 30 s, pour out the waste liquid from the collection tube, and then place the column CP3 into the collection tube.
							</p>
							<p>7. Add 300 μl of rinse solution PWT to the adsorbent column CP3, centrifuge at 12,000 rpm for 30 s, and pour out the waste solution in the collection tube.
							</p>
							<p>8. Place the adsorption column CP3 into the collection tube and centrifuge it at 12,000 rpm for 1 min to remove the rinse solution remaining on the adsorption column.
							</p>
							<p>9. Place the adsorbent column CP3 in a clean centrifuge tube, add 50-100 μl of elution buffer TB dropwise to the middle of the adsorbent membrane, centrifuge at 12,000 rpm for 30 s, and collect the plasmid solution into the centrifuge tube.
							</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 八 -->
			<div id="references8" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d8" aria-expanded="false"
						aria-controls="references-d8" onclick="myFunction(this)">
						<h1 class="accordion-general-label">8. Plasmid transformation coating and inoculation</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d8" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p><b>Heat-excited transformation:</b>Make a sufficient amount of ice into a foam box as an ice bath, take the plasmid, receptive ep tubes containing 50 μL of E. coli and place them on the ice. Add 5μl of plasmid to the receptor ep tube and leave it on ice for 30 min. start the water bath and set the temperature to 42°C. Take out the receptor ep tube from the ice bath and heat stress it. Remove the sensory ep tubes from the ice bath, heat-stimulate for 60 to 90 s, and immediately place back on ice to cool. after 2 min, add 500 μl of liquid LB.
							</p>
							<p><b>Coating plate: </b>After the addition of LB, the receptor ep tubes were placed on a suspension plate and shaken in a shaker shaker at 37°C, 180 rpm, for 40 min. and then centrifuged in a centrifuge at 6000 rpm for 1 min. Take the sensory ep tube and the screening plate containing antibiotics, pipette gun on the ultra-clean bench, aspirate and discard 500 μl of the liquid in the sensory ep tube, the remaining liquid was blown with a pipette gun and then carefully poured into the LB Petri dish, the liquid was evenly coated with a blotting rod and the dish was capped, and then inverted in the incubator overnight. Observe the bacterial growth status on the next day, and if the growth status is good, put it into 4℃ refrigerator for storage.
							</p>
							<p><b>Bacterial inoculation: </b>Take the LB liquid medium containing corresponding antibiotics (5ml per tube) and the bacterial plate in 4℃ refrigerator into the ultra-clean table, take metal tweezers, fire-roasted and sterilized, clip a small gun tip, zap it into a single colony of LB petri dish to pick bacteria, and put it into the LB liquid medium vertically. Put the small test tube with culture medium into the shaker for incubation, 37℃, 180 rpm.
							</p>
							<p><b>Points of Attention: </b>Avoid touching the bottom of the sensory ep tube with hands as much as possible during the whole process; the empty LB petri dish needs to be put in the incubator at 37℃ for drying if there is moisture before picking up the bacteria; the use of ultra-clean table should be sterilized 30min in advance.
							</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 九 -->
			<div id="references9" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d9" aria-expanded="false"
						aria-controls="references-d9" onclick="myFunction(this)">
						<h1 class="accordion-general-label">9. Expansion and IPTG induction</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d9" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p><b>Expansion of bacteria:</b>Prepare 500 ml of LB in a conical flask sterilizer and sterilize it, after cooling, add the corresponding concentration of antibiotics, take the LB medium that was incubated overnight with the bacteria, and pour 5 ml of the bacterial solution. Seal the conical flask and fix it with rubber band. Place the conical flask in a shaker, incubate at 37℃ 180 rpm until the OD value is between 0.6 and 0.8 (turbid to the point that you can't see your fingers through the bacterial liquid). Activate the shaker refrigeration key to reduce the temperature of the shaker thermostatic oscillator to 10℃.</p>
							<p><b>IPTG induction:</b>Add IPTG mother liquor in a conical flask to make the system IPTG concentration of 0.5 mM, put it into the shaker constant temperature oscillator at 10℃, and take it out for observation after 16h.</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 十 -->
			<div id="references10" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d10" aria-expanded="false"
						aria-controls="references-d10" onclick="myFunction(this)">
						<h1 class="accordion-general-label">10. Validation of the construction of osmotic pressure gradient for LB medium in two-component system</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d10" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<h3>(1) Pre-experiment: Determination of the appropriate osmolality range:</h3>
							<table class="model">
								<tbody>
									<tr style="background: rgb(33, 115, 192)">
										<th></th>
										<th>Tryptone</th>
										<th>Yeast extract</th>
										<th>Purified water</th>
										<th>NaCl</th>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>1M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>11.68g</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>2M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>23.36g</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>3M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>35.04g</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>4M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>46.72g</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>5M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>58.40g</td>
									</tr>
								</tbody>
							</table>

							<h3>(2) Formal experiment: validation of hyperosmotic activation of two-component system:</h3>
							<table class="model">
								<tbody>
									<tr style="background: rgb(33, 115, 192)">
										<th></th>
										<th>Tryptone</th>
										<th>Yeast extract</th>
										<th>Purified water</th>
										<th>NaCl</th>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>0M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>0g</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>0.17M (Normal LB)</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>2.00g</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>0.25M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>2.92g</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>0.5M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>5.84g</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>0.75M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>8.76g</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>1M</td>
										<td>2g</td>
										<td>1g</td>
										<td>200ml</td>
										<td>11.68g</td>
									</tr>
								</tbody>
							</table>
						</div>
					</div>
				</div>
			</div>

			<!-- 十一 -->
			<div id="references11" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d11" aria-expanded="false"
						aria-controls="references-d11" onclick="myFunction(this)">
						<h1 class="accordion-general-label">11. Growth curve measuring by enzyme-labeled instrument</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d11" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<h2>(1) Pretreatment of 96-well plates and lids</h2>
							<p>1. prepare 10% Triton X-100 in ethanol by mixing 1 ml of Triton X-100 with 9 ml of ethanol.
							</p>
							<p>2. Pour 10 ml of diluted Triton X-100 (10% Triton X-100) into a new microplate lid.</p>
							<p>3. Incubate the lid with diluted Triton X-100 for 15 s at room temperature.</p>
							<p>4. Take a new 96-well plate and fill each well with 100% ethanol (400 μL per well). Incubate at room temperature for 15min
								incubate at room temperature for 15min.</p>
							<p>5. Discard ethanol from the wells of the microplate and discard Triton X-100 from the microplate lid.</p>
							<p>6. Air-dry the wells and lids in an ultra-clean bench for 30 min.</p>
							<p>7. Turn on the ultra-clean UV lamp to further sterilize the microplate wells and lids for 15min.</p>

							<br>
							<h2>(2) Sample test</h2>
							<table class="model">
								<tbody>
									<tr style="background: rgb(33, 115, 192)">
										<th>Plasmids</th>
										<th>Sodium butyrate(mM)</th>
										<th>Bacteria solution(μl)</th>
										<th>Total volume(μl)</th>
										<th>Replicates</th>
										<th>Resistance</th>
									</tr>

									<tr style="background: rgb(255, 255, 255)">
										<td>[pet28a]-Pompc-eGFP</td>
										<td>0</td>
										<td>5</td>
										<td>200</td>
										<td>3</td>
										<td>KN</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>[pet28a]-Pompc-eGFP</td>
										<td>0.17</td>
										<td>5</td>
										<td>200</td>
										<td>3</td>
										<td>KN</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>[pet28a]-Pompc-eGFP</td>
										<td>0.25</td>
										<td>5</td>
										<td>200</td>
										<td>3</td>
										<td>KN</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>[pet28a]-Pompc-eGFP</td>
										<td>0.5</td>
										<td>5</td>
										<td>200</td>
										<td>3</td>
										<td>KN</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>[pet28a]-Pompc-eGFP</td>
										<td>0.75</td>
										<td>5</td>
										<td>200</td>
										<td>3</td>
										<td>KN</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>[pet28a]-Pompc-eGFP</td>
										<td>1</td>
										<td>5</td>
										<td>200</td>
										<td>3</td>
										<td>KN</td>
									</tr>
								</tbody>
							</table>
							<!-- <p>1. Add 200μL of the sample to be tested into the wells, and set up 3 replicate wells for each sample.</p>
							<p>2. Add 200μL of blank LB to the blank wells as a blank control.</p> -->

							<br>
							<h2>(3) Enzyme marker and software settings</h2>
							<p>1. Turn on the microplate reader and open the Gen5 software.</p>
							<p>2. Adjust the parameters and measure the bacterial growth curve.</p>
							<p>3. Plate type: 96-well plate. b. Lid use: Check.</p>
							<p>4. Lid use: checked. </p>
							<p>5. Temperature: 37°C. </p>
							<p>6. Shake: Track 0:10.</p>
							<p>7. Total time required (e.g., 16 hours, 24 hours, or 48 hours) and kinetic measurements at 10-minute intervals with continuous orbital shaking.</p>
							<p>8. Readings: Absorption at 600 nm.</p>

							<br>
							<h2>(4) Organize the results</h2>
							<p>1. Select the data type as OD600 to observe the results.</p>
							<p>2. After completing data collection, remove the 96-well plate from the microplate reader.</p>
							<p>3. Sterilize the sample and discard the microplate properly.</p>
							<p>4. save the results in the Gen5 software and export the data to an Excel spreadsheet file. 5. close the Gen5 software and export the data to an Excel spreadsheet file.</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 十二 -->
			<div id="references12" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d12" aria-expanded="false"
						aria-controls="references-d12" onclick="myFunction(this)">
						<h1 class="accordion-general-label">12. Fluorescence and luminescence intensity measuring by enzyme-labeled instrument</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d12" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<h2>(1) Pretreatment of 96-well plates and lids</h2>
							<p>1. prepare 10% Triton X-100 in ethanol by mixing 1 ml of Triton X-100 with 9 ml of ethanol.</p>
							<p>2. Pour 10 ml of diluted Triton X-100 (10% Triton X-100) into a new microplate lid.</p>
							<p>3. Incubate the lid with diluted Triton X-100 for 15 s at room temperature.</p>
							<p>4. Take a new 96-well plate and fill each well with 100% ethanol (400 μL per well). Incubate at room temperature for 15min
								incubate at room temperature for 15min.</p>
							<p>5. Discard ethanol from the wells of the microplate and discard Triton X-100 from the microplate lid.</p>
							<p>6. Air-dry the wells and lids in an ultra-clean bench for 30 min.</p>
							<p>7. Turn on the ultra-clean UV lamp to further sterilize the microplate wells and lids for 15min.</p>

							<br>
							<h2>(2) Sample test</h2>
							<p>1. Add 200μL of the sample to be tested into the wells, and set up 3 replicate wells for each sample.</p>
							<p>2.  Add 200μL of blank LB to the blank wells as a blank control.</p>
							
							<br>
							<h2>(3) Enzyme marker and software settings</h2>
							<p>1. Turn on the microplate reader and open the Gen5 software.</p>
							<p>2. Adjust the parameters and measure the bacterial growth curve.</p>
							<p>3. Plate type: 96-well plate. b. Lid use: Check.</p>
							<p>4. Lid use: checked.</p>
							<p>5. Temperature: 37°C.</p>
							<p>6. Shake: Track 0:10.</p>
							<p>7. Total time required (e.g., 16 hours, 24 hours, or 48 hours) and kinetic measurements at 10-minute intervals with continuous orbital shaking.</p>
							<p>8. Readings: Absorption at 600 nm；luminescence at 490nm.</p>

							<br>
							<h2>(4) Organize the results</h2>
							<p>1. Select the data type as OD600 and luminescence to observe the results.</p>
							<p>2. After completing data collection, remove the 96-well plate from the microplate reader.</p>
							<p>3. Sterilize the sample and discard the microplate properly.</p>
							<p>4. save the results in the Gen5 software and export the data to an Excel spreadsheet file. 5. close the Gen5 software and export the data to an Excel spreadsheet file.</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 十三 -->
			<div id="references13" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d13" aria-expanded="false"
						aria-controls="references-d13" onclick="myFunction(this)">
						<h1 class="accordion-general-label">13. SDS-PAGE</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d13" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>
								<b>Protein sample preparation: </b>
								After shaking the amplified bacterial solution well, take 1mL in a centrifuge tube and centrifuge at 10,000 rpm for 10 min. remove the supernatant, leaving about 50ul or so, add a small amount of pH 7.4 PBS to wash three times, leaving about 50ul. Add 50uL 1×Loading Buffer, 0.31g DTT and shake to suspend the precipitate. Place the centrifuge tube in a water bath at 100°C for 10 min. centrifuge at 12,000 rpm for 5-10 min, and the supernatant is the extracted protein. Transfer the supernatant to a new centrifuge tube and add a small amount of Loading Buffer (about 5ul) for sampling.
							</p>
							<p>
								<b>SDS-PAGE gel electrophoresis: </b>
								After shaking the amplified bacterial solution well, take 1mL in a centrifuge tube and centrifuge at 10,000 rpm for 10 min. remove the supernatant, leaving about 50ul or so, add a small amount of pH 7.4 PBS to wash three times, leaving about 50ul. Add 50uL 1×Loading Buffer, 0.31g DTT and shake to suspend the precipitate. Place the centrifuge tube in a water bath at 100°C for 10 min. centrifuge at 12,000 rpm for 5-10 min, and the supernatant is the extracted protein. Transfer the supernatant to a new centrifuge tube and add a small amount of Loading Buffer (about 5ul) for sampling.
							</p>
							<p>
								<b>Staining and decolorization: </b>
								After electrophoresis, recycle the electrophoresis solution, pry open the glass plate, remove the gel into the box and spread it flat. Pour the Caulmers Brilliant Blue staining solution into the cassette, remove it after microwave heating for 2min, and decolorize it with ethanol-acetic acid decolorizing solution, and change the decolorizing solution every 30min until the bands were clearly seen. Pour off the decolorizing solution and observe the bands under a bright background.
							</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 十四 -->
			<div id="references14" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d14" aria-expanded="false"
						aria-controls="references-d14" onclick="myFunction(this)">
						<h1 class="accordion-general-label">14. Preparation and recovery of freeze-dried bacterial powder</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d14" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>1. incubate overnight on LB medium at 37°C with shaking (OD≈1-1.5)</p>
							<p>2. Centrifuge 1 ml of bacterial suspension (12000r, 2 min) and discard supernatant.</p>
							<p>3. Resuspend the bacterial pellet in 500µL of 10% sucrose LB medium.</p>
							<p>4. Transfer to EP tubes and seal with sealing film (holes punched in film to equalize air pressure).</p>
							<p>5. Gradient freezing pretreatment: -4℃ for 30min, -20℃ for 30min, -80℃ for 1h.</p>
							<p>6. Start the lyophilizer to pre-freeze for 2 hours, let the chamber temperature drop to -50°C</p>
							<p>7. Sublimate under vacuum for 12 hours</p>
							<p>8. Increase the temperature to about 30°C to desorb the residual water under vacuum. The lyophilized powder can be stored at -20°C for a long time.</p>
							<p>9. Resuscitation: add 1ml of LB medium into the EP tube to dissolve the powder completely.</p>
							<p>10. Determine the growth curve of resuscitated bacterial powder by using microplate reader.</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 十五 -->
			<div id="references15" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d15" aria-expanded="false"
						aria-controls="references-d15" onclick="myFunction(this)">
						<h1 class="accordion-general-label">15. RecO, RecF anti-UV radiation damage effect verification experiment</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d15" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<h2>(1)Colony plate counting</h2>
							<p>1. Single colonies of recO and recF positive strains and single colonies of E.coli BL21(DE3) containing the pET30b null plasmid and wild-type were picked and incubated overnight at a final concentration of 50 μg/ml kana 5 ml LB culture medium at 37℃, 180 r/min with shaking bed shaking.</p>
							<p>2. Remove the four strains of the overnight culture from the shaker, and dispense them into four autoclaved 15 ml centrifuge tubes, centrifuge for 3 min at 8,000 r/min at room temperature, and discard the supernatant. After adding 10 ml of 0.01M PBS to resuspend the precipitate, centrifuge again at 8,000 r/min for 3 min at room temperature and discard the supernatant.</p>
							<p>3. Add 10 ml PBS to adjust the OD600 of each bacterium close to 1 (spectrophotometer or microplate reader detection)</p>
							<p>4. Take 6 sterile test tubes, numbered 10-1, 10-2, 10-3, 10-4, 10-5, 10-6, take 7 LB solid medium for each strain of bacteria, numbered 10-5, 10-6, and make 3 duplicate plates for each dilution concentration, and leave one plate as blank control. Pipette 4.5 ml of sterile saline into the above numbered tubes with a 5 ml pipette in aseptic operation.</p>
							<p>5. Shake the bacterial solution well, then use a 1 ml sterile pipette to blow back and forth several times in the original sample to be diluted, and then accurately pipette 0.5 ml of bacterial solution into the test tube labeled 10-1. Take another 1 ml sterile pipette, in the same way, first blow the sample back and forth several times in the test tube labeled 10-1 and accurately pipette 0.5 ml of the bacterial solution into the test tube labeled 10-2, and so on until the dilution in the test tube labeled 10-6.</p>
							<p>6. Use 1 ml sterile pipette to suck 0.1 ml of 10-5 and 10-6 diluted bacterial solution accurately, add it to the corresponding numbered LB solid medium, spread it evenly with a pasteurized tube, and put it into 37℃ constant temperature incubator for incubation.</p>
							<p>7. Remove the plates after 24 h of incubation, select the petri dishes with colony numbers within the range of 3-300/plate, calculate the number of colonies on each plate, and calculate the results according to the formula (the number of viable bacteria (pcs/ml) = [(χ1 + χ2 + χ3)/3] × 10 × the number of dilutions).</p>


							<h2>(2)Effect of UV light on bacterial survival</h2>
							<p>1. Take the bacterial solution with adjusted OD and counted, 12,000 r/min, centrifuge for 2 min, discard the supernatant. Resuspend the bacterial solution in 1 ml PBS, and mix the bacteria with sufficient shaking.</p>
							<p>2. Preheat the UV lamp; adjust the irradiation power of the UV lamp and the irradiation table to 1 J/s, and turn on the UV lamp to preheat for 30 min before irradiation to stabilize the intensity of the UV lamp.</p>
							<p>3. For each strain of bacteria, take 25 pieces of LB solid culture medium (so that the final concentration of IPTG in the medium is 0.1 mM), add 0.1 ml of bacterial solution to each medium, and spread it evenly on the surface of the medium with an L stick. The UV irradiation dose (25 J/m2, 50 J/m2, 75 J/m2, 100 J/m2, 125 J/m2, 150 J/m2, 175 J/m2, 200 J/m2) was labeled on the surface of the culture plate, and three replicate plates were made for each irradiation dose, and one plate was left as a blank control.</p>
							<p>4. Start irradiation according to the irradiation dose marked on each plate. After irradiation, all the culture plates were wrapped with black paper, inverted and placed in 37℃ constant temperature incubator, and then taken out and counted on the 3rd day.</p>
						</div>
					</div>
				</div>
			</div>

			<!-- 十六 -->
			<div id="references16" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d16" aria-expanded="false"
						aria-controls="references-d16" onclick="myFunction(this)">
						<h1 class="accordion-general-label">16. Culture of Vibrio Fischeri (prepared with 2216E medium)</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d16" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<table class="model">
								<tbody>
									<tr style="background: rgb(33, 115, 192)">
										<td>Peptone</td>
										<td>5.0</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>Yeast powder</td>
										<td>1.0</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>Ferric citrate</td>
										<td>0.1</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>Sodium Chloride</td>
										<td>19.45</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>Magnesium Chloride</td>
										<td>5.98</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>sodium sulfate</td>
										<td>3.24</td>
									</tr>

									<tr style="background: rgb(219,227,244)">
										<td>calcium chloride</td>
										<td>1.8</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>dicalcium phosphate</td>
										<td>0.55</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>sodium carbonate</td>
										<td>0.16</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>potassium bromide</td>
										<td>0.08</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>strontium chloride</td>
										<td>0.034</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>boric acid</td>
										<td>0.022</td>
									</tr>

									<tr style="background: rgb(219,227,244)">
										<td>sodium silicate</td>
										<td>0.004</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>sodium fluoride</td>
										<td>0.0024</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>ammonium nitrate</td>
										<td>0.0016</td>
									</tr>
									<tr style="background: rgb(255, 255, 255)">
										<td>disodium hydrogenphosphate</td>
										<td>0.008</td>
									</tr>
									<tr style="background: rgb(219,227,244)">
										<td>pH: 7.6±0.2</td>
										<td>25℃</td>
									</tr>
								</tbody>
							</table>
							<p>Weigh 37.4 g of this product, heat and dissolve in 1000 ml of distilled water, 121 ℃ autoclave sterilization for 15 minutes, standby.</p>
							<p>Note: The medium has more inorganic salt components and is prone to precipitation after autoclaving, dissolution before sterilization is critical, add water first, add dry powder slowly while stirring, don't let the dry powder form a ball; after sterilization, it should be shaken well, try to let the precipitation disperse, avoid gathering together to affect the experiment.</p>
							<p>Inoculate with the Vibrio fischeri and place in aerobic culture at 20-25°C for 40-72 hours.</p>
						</div>
					</div>
				</div>
			</div>


		</div><!--------d1dediv----->










		<div id="d2" style="display:none">



			<div id="references15" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d15" aria-expanded="false"
						aria-controls="references-d15" onclick="myFunction(this)">
						<h1 class="accordion-general-label">1.Vibrio fischeri culture</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d15" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>The 2216E medium was used to recover Vibrio fischeri lyophilized powder. The medium inoculated with Vibrio fischeri was incubated at 22℃ for 48h, and the luminescence test was carried out using a multifunctional microplate reader after observing the turbidity of the medium and the blue fluorescence in the darkness with the naked eye.
							</p>
							<div class="row mx-auto d-flex justify-content-center wi-lg-100"><img
									src="https://static.igem.wiki/teams/5210/result-1.png" alt="Gut-sweeper"
									style="width:90%;">
							</div>
							<figcaption class="pt-3 ps-3"><center><strong>Fig 1.</strong><b>Vibrio fischeri successfully cultured and fluoresced in the laboratory</b></center><br>
								A.2216E Culture Media Formulation;<br>
								B.Original camera shot luminescent Vibrio fisheri;<br>
								C.Multifunctional microplate reader luminescence test.<br>
							</figcaption>
                           <p>We successfully cultured Vibrio fischeri in the lab and observed bioluminescence. However, because of the slow growth rate and the time required for luminescence, we did not consider Vibrio fischeri to be suitable as our chassis bacteria, and its luminescence principle inspired us to explore further.</p>
						</div>
					</div>
				</div>
			</div>

			<div id="references16" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d16" aria-expanded="false"
						aria-controls="references-d16" onclick="myFunction(this)">
						<h1 class="accordion-general-label">2.Envz-OmpR Validating of the Two-component osmotic pressure sensing system Envz-OmpR</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d16" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>Validation experiments were performed with BL21(DE3). LB medium with different NaCl concentrations was prepared and reporter gene expression was observed.
								</p>
							<div class="row mx-auto d-flex justify-content-center wi-lg-100"><img
									src="https://static.igem.wiki/teams/5210/result-2.png" alt="Gut-sweeper"
									style="width:90%;"></div>
							<figcaption class="pt-3 ps-3"><center><strong>Fig 2.</strong><b>Fluorescence Intensity with Different osmotic pressure LB cocultured in Pompc-EGFP engineered bacteria</b></center><br>
								A.plasmid construction; <br>
								B.pre-experiment to explore the suitable growth concentration of experimental bacterium BL21; <br>
                                C. fluorescence intensity of engineered bacteria under different osmotic pressures; <br>
								D. Fluorescence microscope to observe the fluorescence intensity of engineered bacteria under different osmotic pressures.<br>
							</figcaption>
                            <p>By observing the intensity of reporter gene expression, we demonstrated the sensitivity of the engineered bacteria to high osmotic pressure.
							</p>
						</div>
					</div>
				</div>
			</div>

			<div id="references17" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d17" aria-expanded="false"
						aria-controls="references-d17" onclick="myFunction(this)">
						<h1 class="accordion-general-label">3.Validating luminescence efficacy of Lux gene clusters</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d17" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>The Lux gene cluster (LuxCDABE or LuxCDABEGF) was introduced into BL21 (DE3) and expression was induced using IPTG. Bioluminescence was observed by naked eye and photographed. After the obvious luminescence was observed by naked eyes, the fluorescence intensity was quantified using a multifunctional microplate reader.
							</p>
							<div class="row mx-auto d-flex justify-content-center wi-lg-100"><img
									src="https://static.igem.wiki/teams/5210/result-3.png" alt="Gut-sweeper"
									style="width:80%;"></div>
							<figcaption class="pt-3 ps-3"><center></strong><b>Fig 3.Luminescence results of optimized luminescent gene circuits</b></center><br>
								A.Plasmid construction;<br>
								B.Multifunctional microplate reader luminescence test; <br>
								C.Original camera shot luminescent engineering bacteria; <br>
								D.Exposure 15s shot luminescent engineering bacteria. <br>
							</figcaption>
                            <p>Lux luminescent gene clusters emit blue fluorescence that is visible to the naked eye, however, it can only be observed in dark environments, and we would like to further enhance the intensity of the luminescence.</p>
						</div>
					</div>
				</div>
			</div>


			<div id="references18" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d18" aria-expanded="false"
						aria-controls="references-d18" onclick="myFunction(this)">
						<h1 class="accordion-general-label">4.Validating cold-tolerant module protein expression	</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d18" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>Plasmids containing proteins encoding SOD and Catalase were introduced into BL21 (DE3) and expression was induced using 0.5 mM and 0 mM IPTG, respectively, and SDS-PAGE was performed to verify protein expression.</p>
							<div class="row mx-auto d-flex justify-content-center wi-lg-100"><img
									src="https://static.igem.wiki/teams/5210/result-4.png"
									alt="Gut-sweeper" style="width:80%;"></div>
							<figcaption class="pt-3 ps-3"><center><strong>Fig 4.</strong><b>Prokaryotic expression of SOD and catalase proteins</b></center><br>
								A.Plasmid construction;<br>
								B.SDS-PAGE image; <br>
								C.SDS-PAGE strip analysis results.<br>
							</figcaption>


							<p>We successfully expressed two proteins, but the amount of expression was not ideal. In the future, we will further explore to optimize the expression and test the engineered bacteria for low temperature resistance.</p>

						</div>
					</div>
				</div>
			</div>

			<div id="references19" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d19" aria-expanded="false"
						aria-controls="references-d19" onclick="myFunction(this)">
						<h1 class="accordion-general-label">5.Preparation and recovery of lyophilized bacterial powder</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d19" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>BL21 (DE3) was lyophilized using liquid LB medium with 10% sucrose as a protective agent. After one week of lyophilization and preservation, the plates were resuscitated using liquid LB medium, diluted 3,000-fold, coated and cultured in microtiter plates. The coated plates and culture results were visualized.</p>
							<div class="row mx-auto d-flex justify-content-center wi-lg-100"><img
									src="https://static.igem.wiki/teams/5210/result-5.png"
									alt="Gut-sweeper" style="width:90%;"></div>


							<figcaption class="pt-3 ps-3"><center><strong>Fig 5.</strong><b>Results of preparation of lyophilized bacterial powders and resuscitation</b></center><br>
								A.Freeze-dried bacterial powder product; <br>
								B.Diluted coated plate result after resuscitation; <br>
								C.Visualization of coated plate result;<br>
								D.Measurement of growth curve after resuscitation.<br>
							</figcaption>
						</div>
					</div>
				</div>
			</div>

			<div id="references20" style="box-shadow: 0 6px 7px -2px rgba(0,0,0,0.5);">
				<div class="row my-5" style="cursor: pointer">
					<button class="btn d-flex justify-content-between accordion-general w-100" type="button"
						data-bs-toggle="collapse" data-bs-target="#references-d20" aria-expanded="false"
						aria-controls="references-d20" onclick="myFunction(this)">
						<h1 class="accordion-general-label">6.Verificating radiation resistance of RecO, RecF</h1>
						<div>
							<div class="container">
								<div class="bar1"></div>
								<div class="bar2"></div>
								<div class="bar3"></div>
							</div>
						</div>
					</button>
					<div class="collapse px-0" id="references-d20" style="margin-top: -10px;">
						<div class="card card-body accordion-general-content px-5">
							<p>Plasmids containing genes encoding RecO and RecF were introduced into BL21 (DE3), diluted coated plates (agar plates containing IPTG) were irradiated with different doses of UV light, and surviving colonies were counted after 3 days of incubation. Protein expression was induced with 1 mM IPTG and SDS-PAGE was performed to verify protein expression. </p>


							<div class="row mx-auto d-flex justify-content-center wi-lg-100"><img
									src="https://static.igem.wiki/teams/5210/result-6.png" alt="Gut-sweeper" style="width:90%;"></div>
							<figcaption class="pt-3 ps-3"><center><strong>Fig 6.</strong><b>Results of prokaryotic expression of RecO and RecF genes and validation of radiation resistance</b></center><br>
								A.Plasmid construction;<br>
								B.Changes in the survival of bacteria after UV irradiation at different doses;<br>
								C.SDS-PAGE image.<br>
							</figcaption>


							<p>We successfully expressed RecO and RecF proteins in BL21(DE3) and verified the survivability of the engineered bacteria with UV irradiation.
							</p>

						</div>
					</div>
				</div>
			</div>









		</div><!-----d2dediv--->






	</div><!--这个是整个container的/div-->

	{% endblock %}